code/mission/missionbriefcommon.cpp

```diff
@@ -417,14 +447,36 @@ void brief_parse_icon_tbl(const char* filename)
 			}
 		}
 		else { // old style
+
+			size_t Num_species_in_table = 0;
+			
+			//Calculate how many species we're going to try to parse icons for using the pre-parsed count
+			if (Num_icons_in_table % MIN_BRIEF_ICONS != 0)
+				Warning(LOCATION,
+					"An incorrect number of icons was found in icons.tbl. There should be %i icons per species listed!",
+					MIN_BRIEF_ICONS);
+
+			Num_species_in_table = Num_icons_in_table / MIN_BRIEF_ICONS;
```

Comment:
Suggestion: use a temporary variable here, because at this intermediate step, `Num_species_in_table` does not actually represent the number of species in the table.

Reply:
Alright. Done.